api/v1/Game.php

```diff
@@ -805,15 +803,11 @@ public function GetGameDetails(): array|PromiseInterface
             }
 
             $realtimePerEra = explode(",", $state["game_planning_era_realtime"]);
-            // todo: division by zero.
-            $currentEra = intval(floor($state["game_currentmonth"] / $state["game_planning_gametime"]));
-            $realtimePerEra[$currentEra] = $state["game_planning_realtime"];
```

Comment:
So this code is obsolete apperently ...

so an array is created from : 
`$realtimePerEra = explode(",", $state["game_planning_era_realtime"]);`
in both the new and old code

But in the old code it overrides one of the array element :
`$realtimePerEra[$currentEra] = $state["game_planning_realtime"];`
where $currentEra is:
`$currentEra = intval(floor($state["game_currentmonth"] / $state["game_planning_gametime"]));`

So this was all overcomplicated , and unnecessary?
You probably stepped to the code, so you have seen the values.  So I assume the creation of the array using the explode already had the correct values, even for "$currentEra" ?

Reply:
has been explained